knative-operator/pkg/apis/addtoscheme_operator_v1alpha1.go

```diff
@@ -1,11 +1,10 @@
 package apis
```

Comment:
This file was adding the upstream operator schemas.
That happens in another file now: [knative-operator/pkg/apis/addtoscheme_knative_operator_v1alpha1.go](https://github.com/openshift-knative/serverless-operator/pull/472/files#diff-00904b3d74ed5c8bf905a77433451d24)

This file is now adding the stuff under `operator.serverless.openshift.io`.